amida-tech/amida-messaging-microservice
# Feature/orange 824 tags for threads

***This is branched off #67 on purpose to instigate that being reviewed and merged. It must be merged before this one.***

#### What's this PR do?
Adds the `tags` column to Thread schema.
Accepts `tags` field in the create thread endpoint (optional).
`tags` is a list of strings.
`tags` are included when you call the threads index endpoint (returns list of all threads).

This allows for filtering client side based on `tags` set when the thread was created.
#### Related JIRA tickets:
Orange-824
#### How should this be manually tested?
Via postman
Create a thread with tags, something like:

```
{
  topic: "topic",
  message: "message",
  participants: ["user1"],
  tags: ["test"]
}
```

Then get the list of all threads. Make sure it includes the tags field with the tags you posted when creating the thread.
#### Any background context you want to provide?
This does add a column to the Thread Schema. For testing the easiest way to update is deleting the old db and recreating, letting sequelize create the schema.
I think thats fine for now since we haven't deployed yet.
#### Screenshots (if appropriate):
